lib/Settings/Settings.cpp

```diff
@@ -29,10 +29,16 @@
 
 #include "Settings.h"
 
+#include <ArduinoJson.h>
```

Comment:
Now we have two Json implementations, the internal of pilight/ESPiLight and ArduinoJson. How about to expose the pilight json with ESPiLight in a similar api like AduinoJson? But still not sure, if this can provide the same performance, e.g. Stream support.

Reply:
ArduinoJson is way way better to use. I think the pilight one is small enough to not worry.

That is also part why I did not use inheritance to Esp-milight-hub, because I didn't not want any types from pilight to "leak" into other compile units.